version/version.go

```diff
@@ -1,5 +1,28 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("version")
+
 var (
-	Version = "0.0.1"
+	Version   = "" // version will be replaced while building the binary using ldflags
+	GoVersion = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
 )
+
+// Print() logs the operator version and related information
+func Print() {
+	log.Info("operator", "version", Version)
+	log.Info("go", "version", GoVersion)
+	log.Info("operator-sdk", "version", sdkVersion.Version)
+}
+
+// Get() returns the operator version
+func Get() string {
```

Comment:
This could be renamed to something more specific GetVersion()

Reply:
cc https://github.com/openshift/windows-machine-config-operator/pull/82#discussion_r456053518